2023-05-07 09:55:30,226 INFO    MainThread:17175 [internal.py:wandb_internal():93] W&B internal server running at pid: 17175, started at: 2023-05-07 09:55:30.226194
2023-05-07 09:55:30,228 INFO    WriterThread:17175 [datastore.py:open_for_write():75] open: /kaggle/working/multi_label/exp17_scaling/wandb/run-20230507_095528-2x8zgt22/run-2x8zgt22.wandb
2023-05-07 09:55:30,230 DEBUG   SenderThread:17175 [sender.py:send():236] send: header
2023-05-07 09:55:30,230 DEBUG   SenderThread:17175 [sender.py:send():236] send: run
2023-05-07 09:55:30,594 INFO    SenderThread:17175 [dir_watcher.py:__init__():216] watching files in: /kaggle/working/multi_label/exp17_scaling/wandb/run-20230507_095528-2x8zgt22/files
2023-05-07 09:55:30,594 INFO    SenderThread:17175 [sender.py:_start_run_threads():818] run started: 2x8zgt22 with start time 1683453329
2023-05-07 09:55:30,595 DEBUG   SenderThread:17175 [sender.py:send():236] send: summary
2023-05-07 09:55:30,596 DEBUG   HandlerThread:17175 [handler.py:handle_request():140] handle_request: check_version
2023-05-07 09:55:30,596 INFO    SenderThread:17175 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-05-07 09:55:30,596 DEBUG   SenderThread:17175 [sender.py:send_request():250] send_request: check_version
2023-05-07 09:55:30,627 DEBUG   HandlerThread:17175 [handler.py:handle_request():140] handle_request: run_start
2023-05-07 09:55:30,640 DEBUG   HandlerThread:17175 [meta.py:__init__():35] meta init
2023-05-07 09:55:30,640 DEBUG   HandlerThread:17175 [meta.py:__init__():49] meta init done
2023-05-07 09:55:30,640 DEBUG   HandlerThread:17175 [meta.py:probe():209] probe
2023-05-07 09:55:30,650 DEBUG   HandlerThread:17175 [git.py:repo():29] git repository is invalid
2023-05-07 09:55:30,650 DEBUG   HandlerThread:17175 [meta.py:_save_code():87] save code
2023-05-07 09:55:30,650 WARNING HandlerThread:17175 [meta.py:_save_code():89] unable to save code -- program entry not found
2023-05-07 09:55:30,650 DEBUG   HandlerThread:17175 [meta.py:_save_pip():53] save pip
2023-05-07 09:55:30,651 DEBUG   HandlerThread:17175 [meta.py:_save_pip():67] save pip done
2023-05-07 09:55:30,652 DEBUG   HandlerThread:17175 [meta.py:_save_conda():74] save conda
2023-05-07 09:55:31,596 INFO    Thread-7  :17175 [dir_watcher.py:_on_file_created():269] file/dir created: /kaggle/working/multi_label/exp17_scaling/wandb/run-20230507_095528-2x8zgt22/files/requirements.txt
2023-05-07 09:55:31,597 INFO    Thread-7  :17175 [dir_watcher.py:_on_file_created():269] file/dir created: /kaggle/working/multi_label/exp17_scaling/wandb/run-20230507_095528-2x8zgt22/files/conda-environment.yaml
2023-05-07 09:55:31,597 INFO    Thread-7  :17175 [dir_watcher.py:_on_file_created():269] file/dir created: /kaggle/working/multi_label/exp17_scaling/wandb/run-20230507_095528-2x8zgt22/files/wandb-summary.json
2023-05-07 09:55:34,758 WARNING MainThread:17175 [internal.py:wandb_internal():153] Internal process interrupt: 1
2023-05-07 09:55:34,961 DEBUG   HandlerThread:17175 [meta.py:_save_conda():84] save conda done
2023-05-07 09:55:34,962 DEBUG   HandlerThread:17175 [meta.py:probe():247] probe done
2023-05-07 09:55:35,062 WARNING MainThread:17175 [internal.py:wandb_internal():153] Internal process interrupt: 2
2023-05-07 09:55:35,062 ERROR   MainThread:17175 [internal.py:wandb_internal():156] Internal process interrupted.
2023-05-07 09:55:35,230 INFO    WriterThread:17175 [datastore.py:close():279] close: /kaggle/working/multi_label/exp17_scaling/wandb/run-20230507_095528-2x8zgt22/run-2x8zgt22.wandb
2023-05-07 09:55:35,259 INFO    HandlerThread:17175 [handler.py:finish():806] shutting down handler
2023-05-07 09:55:35,596 INFO    Thread-7  :17175 [dir_watcher.py:_on_file_created():269] file/dir created: /kaggle/working/multi_label/exp17_scaling/wandb/run-20230507_095528-2x8zgt22/files/wandb-metadata.json
2023-05-07 09:55:35,625 INFO    SenderThread:17175 [sender.py:finish():1114] shutting down sender
2023-05-07 09:55:35,626 INFO    SenderThread:17175 [dir_watcher.py:finish():353] shutting down directory watcher
